WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -268,4 +268,33 @@ def pdf_name(competition)
   def cached_pdf_name(competition, colors)
     "#{pdf_name(competition)}_#{competition.updated_at.iso8601}_#{colors}"
   end
+
+  def registration_status_icon(competition)
+    icon = ""
+    title = ""
+    icon_class = ""
+
+    if competition.registration_not_yet_opened?
+      icon = "clock"
+      title = I18n.t('competitions.index.tooltips.registration.opens_in', duration: distance_of_time_in_words_to_now(competition.registration_open))
+      icon_class = "blue"
+    elsif competition.registration_past?
+      icon = "user times"
+      title = I18n.t('competitions.index.tooltips.registration.closed')
+      icon_class = "red"
+    elsif competition.registration_full?
+      icon = "user"
```

Comment:
What about using `user clock` here? I'm a bit at odds with the fact that both green and red have a "user plus symbol" icon but orange doesn't.